scripts/zones/Altar_Room/Zone.lua

```diff
@@ -12,6 +12,9 @@ end;
 
 function onZoneIn(player,prevZone)
     local cs = -1;
+    if (player:getVar("FickblixCS") == 1) then
```

Comment:
In the majority of the code, you removed parenthesis, but added them here. No need for inner/outer parenthesis in these conditionals.